Typing.d.ts

```diff
@@ -48,7 +48,7 @@ export type Tenant = {
 	country: string;
 	guarantor_name: string
 	guarantor_phone: string
-	guarantor_address: string
+	guarantor_address: string    
```

Comment:
what's this space doing here?
Please review your changes properly before commiting

Reply:
Another on #131